skyreader/plot/plot.py

```diff
@@ -617,67 +903,122 @@ def bounding_box(ra, dec, theta, phi):
+                np.degrees(extra_lat),
+                lonlat=True,
+                c='m',
+                marker='x',
+                s=20,
+                label=r'Reported online (50%, 90%)'
+            )
+            for cont_scale, cont_col, cont_sty in zip(
+                    [1., 2.1459/1.177], ['m', 'm'], ['-', '--']
+            ):
+                spline_contour = self.circular_contour(
+                    extra_ra_rad,
+                    extra_dec_rad,
+                    extra_radius_rad*cont_scale,
+                    healpy.get_nside(equatorial_map)
+                )
                 spline_lon = spline_contour[1]
                 spline_lat = np.pi/2. - spline_contour[0]
-                healpy.projplot(np.degrees(spline_lon), np.degrees(spline_lat), 
-                    lonlat=True, linewidth=2., color=cont_col, 
-                    linestyle=cont_sty)
+                healpy.projplot(
+                    np.degrees(spline_lon),
+                    np.degrees(spline_lat),
+                    lonlat=True,
+                    linewidth=2.,
+                    color=cont_col,
+                    linestyle=cont_sty
+                )
 
         plt.legend(fontsize=6, loc="lower left")
 
-
-        print("Contour Area (50%):", contour_areas[0], "degrees (scaled)")
-        print("Contour Area (90%):", contour_areas[1], "degrees (scaled)")
-
-
         # Dump the whole contour
-        pickle_path = self.output_dir / (unique_id + ".contour.pkl")
-        LOGGER.info(f"Saving contour to {pickle_path}")
-        with open(pickle_path, "wb") as f:
+        path = unique_id + ".contour.pkl"
+        print("Saving contour to", path)
+        with open(path, "wb") as f:
             pickle.dump(saving_contours, f)
 
-        healpy.write_map(self.output_dir / f"{unique_id}.skymap_nside_{mmap_nside}.fits.gz",
-            equatorial_map, coord = 'C', column_names = ['2LLH'],
-            extra_header = fits_header, overwrite=True)
+        # logic for multiordermaps -> for future developements
+
+        # save multiorder version of the map
+        # multiorder_map = mhealpy.HealpixMap(
+        #     grid_value, uniq_array
+        # )
+        # multiorder_map.write_map(
+        #     f"{unique_id}.skymap_nside_{mmap_nside}.multiorder.fits.gz",
+        #     column_names=["PROBABILITY"],
+        #     extra_header=fits_header,
+        #     overwrite=True,
+        # )
+
+        if llh_map:
+            column_names = ['2LLH']
+        else:
+            # avoid excessively heavy data format for the flattened map
+            equatorial_map = equatorial_map.clip(
+                1.e-16, None
+            ).astype('float64')
+            # renormalize
+            equatorial_map = equatorial_map / np.nansum(equatorial_map)
+            column_names = ["PROBABILITY"]
+
+        # save flattened map
+        healpy.write_map(
+            f"{unique_id}.skymap_nside_{mmap_nside}.fits.gz",
```

Comment:
it would be nice to have some indicator in the filename/a slightly different naming convention if this is a probability map vs a deltallh one (even though it's also in the header, it would be nice to know what type of map it is at a glance without needing to open the file)

Reply:
I added now an indicator in the last commit